src/community/wfs3/src/main/java/org/geoserver/wfs3/kvp/ResultTypeKvpParser.java

```diff
@@ -0,0 +1,28 @@
+/* (c) 2019 Open Source Geospatial Foundation - all rights reserved
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
+ */
+package org.geoserver.wfs3.kvp;
+
+import net.opengis.wfs20.ResultTypeType;
+import org.geoserver.ows.KvpParser;
+import org.geotools.util.Version;
+
+/**
+ * ResultType parser for WFS 3.0.
+ *
+ * @author Fernando Mino - Geosolutions
+ */
+public class ResultTypeKvpParser extends KvpParser {
```

Comment:
I believe the gs-wfs ResultTypeKvpParser should already be covering this, is there a need to roll a custom WFS 3 one?